.laminas-ci.json

```diff
@@ -0,0 +1,5 @@
+{
+    "extensions": [
+         "pdo-sqlite",
```

Comment:
Maybe GHA is fails due to the trailing `,` but not 100% sure about it, might be only the empty lines in `composer.json` tho.